src/site/content/en/lighthouse-performance/first-contentful-paint/index.md

```diff
@@ -46,24 +47,20 @@ This table shows how to interpret your FCP score:
       <tr>
         <th>FCP time<br>(in seconds)</th>
         <th>Color-coding</th>
-        <th>FCP score<br>(HTTP Archive percentile)</th>
       </tr>
     </thead>
     <tbody>
       <tr>
-        <td>0–2</td>
+        <td>0–1.8</td>
         <td>Green (fast)</td>
-        <td>75–100</td>
```

Comment:
these have been removed on the newer metric pages (e.g. https://web.dev/lighthouse-total-blocking-time/#how-lighthouse-determines-your-tbt-score) and Lighthouse doesn't expose the per-metric score directly (just in the raw JSON response), so it seemed good to remove them (otherwise they need to be updated to 90-100)